e2e/cypress/integration/emoji/custom_emoji_spec.js

```diff
@@ -34,13 +80,520 @@ describe('Custom emojis', () => {
+
+            // # Filter out the url from the css background property
+            // url("https://imageurl") => https://imageurl
+            const url = imageSpan.css('background-image').split('"')[1];
+
+            // # Verify that the emoji image i sthe correct one
+            cy.fixture(largeEmojiFile).then((overrideImage) => {
+                cy.request({url, encoding: 'base64'}).then((response) => {
+                    expect(response.status).to.equal(200);
+                    expect(response.body).to.eq(overrideImage);
+                });
+            });
+        });
+
+        // # User1 logs in
+        cy.apiLogin(testUser);
+
+        // # The emoji should be displayed in the post
+        // * Get the emoji image span
+        cy.getLastPost().find('p').find('span > span').then((imageSpan) => {
+            cy.expect(imageSpan.attr('title')).to.equal(customEmojiWithColons);
+
+            // # Filter out the url from the css background property
+            // url("https://imageurl") => https://imageurl
+            const url = imageSpan.css('background-image').split('"')[1];
+
+            // # Verify that the emoji image i sthe correct one
+            cy.fixture(largeEmojiFile).then((overrideImage) => {
+                cy.request({url, encoding: 'base64'}).then((response) => {
+                    expect(response.status).to.equal(200);
+                    expect(response.body).to.eq(overrideImage);
+                });
+            });
+        });
+
+        deleteCustomEmoji(testTeam);
+    });
+
+    it('MM-T2186 Emoji picker - default and custom emoji reaction, case-insensitive', () => {
+        const messageText = 'test message';
+
+        // # Open sidebar
+        cy.get('#sidebarHeaderDropdownButton').click();
+
+        // # Click on custom emojis
+        cy.get('#customEmojis').should('be.visible').click();
+
+        // # Click on add new emoji
+        cy.get('.btn-primary').click();
+
+        // # Type emoji name
+        cy.get('#name').type(customEmojiWithColons);
+
+        // # Select emoji image
+        cy.get('input#select-emoji').attachFile(largeEmojiFile);
+
+        cy.get('.backstage-form__footer').within(($form) => {
+            // # Click on Save
+            cy.wrap($form).find('.btn-primary').click();
+        });
+
+        // # Navigate to a channel
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Post a message
+        cy.postMessage(messageText);
+
+        // # Edit post and add a message
+        cy.getLastPostId().then((postId) => {
+            const postText = `#postMessageText_${postId}`;
+            cy.get(postText).should('have.text', messageText);
+
+            // # Press the up arrow to open the edit modal
+            cy.get('#post_textbox').type('{uparrow}');
+
+            // # Search for the pre-built emoji
+            // * Show emoji list
+            cy.get('#editPostEmoji').should('be.visible').click();
```

Comment:
Step 4 is not via editing a post but reacting to a post -- "React again to that post by clicking emoji picker, typing the first three letters of the custom emoji name in search, and clicking the custom emoji". May open reaction to a post by,
```javascript
cy.clickPostReactionIcon(postId);
```
<img width="113" alt="Screen Shot 2020-10-08 at 8 02 33 PM" src="https://user-images.githubusercontent.com/5334504/95456020-452e9f00-09a1-11eb-9829-8d0c47331fd7.png">